styles.css

```diff
@@ -159,23 +159,24 @@ a[href*="google"]:link {
 
 .data{
   background-color: #FFFFFF;
-  width:22vw;
-  height: 53vh;
+  width:24em;
+  height: 45em;
   text-align: left;
   padding:1.2em 1.1em;
+  display:block;
```

Comment:
All divs have `display:block` set by default, you should not need to add it here.